### ISSUE TITLE
Tab order does not work as expected

### ISSUE THREAD

**Issue by P5-133XL** [FoldingAtHome/fah-nacl-client#21]
When in a text box (username, team number, passkey) typing the tab-key does not cycle through the other text boxes but goes to a link then to text box to link to text box ...
